go/chat/flipmanager.go

```diff
@@ -704,10 +704,14 @@ func (m *FlipManager) parseMultiDie(arg string, nPlayersApprox int) (start flip.
 	return flip.NewStartWithBigInt(m.clock.Now(), val, nPlayersApprox), nil
 }
 
+const shuffleSeparaters = ",，"
```

Comment:
is there a space in there?

Reply:
Nuh; Unicode commas come with built-in spacing for free 😉

Reply:
bump!